EPRN.Common/Constants/Constants.cs

```diff
@@ -102,7 +102,14 @@ public static class Waste
                 public const string ReProcessorExport = "ReProcessorExport";
                 public const string Note = "Note";
                 public const string DecemberWaste = "DecemberWaste";
+                public const string AccredidationLimit = "AccredidationLimit";
             }
         }
     }
+}
+
+public static class Double
+{
+    public static readonly double AccredidationLimit = 400;
```

Comment:
Should this not be configurable?

Reply:
According to Kate, this is a fixed figure and will not change. If/when that situation changes we can move it to a config file.